bin/gce-post-align.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -o pipefail
+
+ZONES[0]="us-central1-a"
+ZONES[1]="us-central1-b"
+ZONES[2]="us-central1-c"
+ZONES[3]="us-central1-f"
+MACHINE_ZONE=${ZONES[$[ $RANDOM % 4 ]]}
+MACHINE_NAME=$(echo "post-align-"$(basename $1) | tr "[:upper:]" "[:lower:]" | sed "s/[^a-z0-9]/-/g" | head -c62)
+#MACHINE_TYPE_OPTS="--machine-type n1-standard-2"
+MACHINE_TYPE_OPTS="--custom-cpu 1 --custom-memory 6656MiB"
+
+gcloud compute instances create --zone $MACHINE_ZONE --scopes storage-full --image ubuntu-1404-alignment $MACHINE_TYPE_OPTS --boot-disk-size 300 $MACHINE_NAME
+
+EXIT_STATUS=$?
+if [[ $EXIT_STATUS == 0 ]]
+then
+  # Give ssh daemon some time to get running.
+  sleep 30s  
+
+  echo "[$(date)] Downloading input crams"
+  START_TIME=$(date +%s)
+  gcloud compute ssh --zone $MACHINE_ZONE $MACHINE_NAME -- gsutil cp $1'/*.cram' /home/alignment/
```

Comment:
The wildcard is needed to only download crams instead of everything that is in that directory.